src/LightPanels/LightPanels.js

```diff
@@ -192,7 +192,6 @@ module.exports = kind(
 			this.updateTransforms();
 			this.orientationChanged();
 			this.directionChanged();
-			this.indexChanged();
```

Comment:
I think we may need something like this in the future, to support specifying a starting index. This isn't really a common use case right now, so I'll open a ticket for follow-on work, and we can figure out a way to work around this.